### PR TITLE
Add ParallaxBanners to Hero images

### PR DESCRIPTION
**Pull request by jfranciscosousa**
Uses `react-scroll-parallax` for the effect.

### CODE REVIEW

File: src/components/layout/index.js [hunk 1]
```diff
@@ -7,16 +7,18 @@ import Footer from "./Footer"
...
-  <div styleName="root">
-    <Helmet>
-      <link rel="stylesheet" href="https://use.typekit.net/bcx8qfd.css" />
-    </Helmet>
-    <Header />
-    <main>{children}</main>
-    <Footer />
-  </div>
-)
+const Layout = ({ children }) => {
```
Comment (pfac) [inline src/components/layout/index.js]: Why this change?

File: src/common/reset.css [hunk 2]
```diff
@@ -36,8 +36,17 @@ address {
...
   position: absolute !important;
-  height: 1px; width: 1px;
+  height: 1px;
+  width: 1px;
   overflow: hidden;
   clip: rect(1px 1px 1px 1px); /* IE6, IE7 */
   clip: rect(1px, 1px, 1px, 1px);
 }
+
+.Parallax-outer-override {
```
Comment (pfac) [inline src/common/reset.css]: Since these classes refer to a specific package, can you please put them in `src/common/react-scroll-parallax.css` and include it in `normalize.css`?

Comment (jfranciscosousa) [reply src/common/reset.css]: Sure, good idea.

File: src/components/layout/index.js (hunk 1 above)
Comment (jfranciscosousa) [reply src/components/layout/index.js]: Nevermind, these are leftovers from some experiences. I'll revert this

**Comment by jfranciscosousa** [comment]
I did not optimize the images, I'll do it. Didn't think about the repo size sorry :stuck_out_tongue:

**Comment by joaojusto** [comment]
Only Pedro does :D
